mock/soap/src/main/java/io/gatehill/imposter/plugin/soap/parser/VersionAwareWsdlParser.kt

```diff
@@ -62,7 +62,10 @@ class VersionAwareWsdlParser(wsdlFile: File) : WsdlParser {
     private val delegate: WsdlParser
 
     init {
-        val document = SAXBuilder().build(wsdlFile)
+        val document = SAXBuilder().apply {
```

Comment:
I wasn't clear if this was actually necessary - the tests pass without this. WDYT?

Reply:
It looks indeed not necessary. Probably JDom is able to handle it correctly (in contrast to XmlBeans).

Better to leave it out then.